lib/fastlane/actions/sigh.rb

```diff
@@ -18,6 +18,7 @@ def self.run(params)
         app = CredentialsManager::AppfileConfig.try_fetch_value(:app_identifier)
         raise 'No app_identifier definied in `./fastlane/Appfile`'.red unless app
 
+        CredentialsManager::PasswordManager.shared_manager(ENV['SIGH_USERNAME']) if ENV['SIGH_USERNAME']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.